cranelift/codegen/shared/src/condcodes.rs

```diff
@@ -122,6 +122,27 @@ impl IntCC {
         }
     }
 
+    /// Determines whether this condcode interprets inputs as signed or
+    /// unsigned.  See the documentation for the `icmp` instruction in
+    /// cranelift-codegen/meta/src/shared/instructions.rs for further insights
+    /// into this.
+    pub fn is_signed(&self) -> bool {
```

Comment:
Would it be safer to panic on any cases that we don't expect to ever get -- are there any?  Specifically, for equality, signfulness is meaningless, and for overflows -- aren't there both signed and unsigned overflows?  Certainly on x64 there are (`rflags.C` vs `rflags.O`).

Reply:
Hmm, reverted this change; this was code motion from the aarch64 code, but your comment made me realize the returned values likely only apply to aarch64, and eventually x64 didn't use it.